crux-core/src/crux/history_options.clj

```diff
@@ -1,36 +1,24 @@
-(ns crux.history-options
-  (:import crux.api.HistoryOptions$SortOrder))
+(ns crux.history-options)
```

Comment:
I don't think we need this in Clojure any more? Would it be worth us considering making `HistoryOptions` a concrete class?